packages/babel-plugin/src/utils/build-compiled-component.ts

```diff
@@ -57,6 +58,49 @@ const getExpression = (
   return value;
 };
 
+/**
+ * Check if 'node' is a 'inBuiltComponent'.
+ * We assume it's a 'inBuiltComponent' if it starts with a lowercase letter (i.e. div).
+ *
+ * @param node Originating node
+ */
+const isInBuiltComponent = (node: t.JSXElement): boolean => {
+  let tagName;
+
+  // Get the tag name (i.e. td) from node <td />
+  if (
+    t.isJSXIdentifier(node.openingElement.name) ||
+    t.isJSXNamespacedName(node.openingElement.name)
+  ) {
+    tagName = node.openingElement.name.name;
+  } else {
+    tagName = node.openingElement.name.property.name;
+  }
+
+  if (typeof tagName !== 'string') return false;
+
+  // Check if the first letter is lower case
+  return tagName.charCodeAt(0) >= 97 && tagName.charCodeAt(0) <= 122;
+};
+
+/**
+ * Check if the `ax` runtime utility is required.
+ * We assume it's required if
+ * - there is a class name prop statically defined
+ * - there is conditional css
+ * - 'node' is a 'UserDefinedComponent'
+ *
+ * @param node Originating node
+ * @param classNames The classNames for a component at runtime
+ *
+ */
+const isAxRequired = (node: t.JSXElement, classNames: t.Expression[]) => {
+  return (
+    isInBuiltComponent(node) &&
```

Comment:
maybe I misunderstood but thought that `ax` is not required in cases such as `<div css={css({color: 'red'})} />`

Reply:
Sorry, I don't quite follow your question. Can you elaborate it please?

Reply:
nvm, I just read the description... I get it now :)